api/src/main/kotlin/com/empowerops/volition/dto/grpc-extensions.kt

```diff
@@ -86,13 +91,13 @@ class LoggingInterceptor(val output: Appendable): ServerInterceptor {
         val inboundInterceptor = object: ForwardingServerCallListener.SimpleForwardingServerCallListener<T>(actual) {
             override fun onComplete() {
                 if( ! type.serverSendsOneMessage()){
-                    output.append("CLOSED: $fullMethodName")
+                    logger.log("$fullMethodName", "API CLOSED")
                 }
                 actual.onComplete()
             }
             override fun onMessage(message: T) {
-                output.appendln("INBOUND: $fullMethodName")
-                output.appendln(message.toString())
+                val messageString = message.toString().let { if(it.isBlank()) "[empty message]" else "\n$it" }
+                logger.log("$fullMethodName $messageString", "API INBOUND")
                 actual.onMessage(message)
```

Comment:
A little note:
This does make our reiser call a bit confusing since they are reply to register call, 
it will show blablabla register() then followed by the response to a evaluation. I am thinking renaming register to register request.